integration/auth/user_mgt_test.go

```diff
@@ -954,3 +1146,23 @@ func importUser(t *testing.T, uid string, userToImport *auth.UserToImport) *auth
 	}
 	return userRecord
 }
+
+// Helper function that deletes the user with the specified phone number if it
+// exists.
+// TODO(rsgowman): This function was ported from node.js port; a number of tests
+// there use this, but haven't been ported to go yet. Do so.
+func deletePhoneNumberUser(t *testing.T, phoneNumber string) {
```

Comment:
We really ought to stop passing `t` to helper functions. Instead return an error from this function, and handle it at the callsite.

Reply:
This pattern comes direct from go/go-style/best-practices#test-helper-error-handling.

I like it, since it reduces clutter at the call site for what's essentially a failed precondition. But I can change it to return an error if you feel strongly.

Reply:
Oh, that's good to know. I guess I'm always a bit weirded out by code that passes `t` around. But I suppose that's considered normal in Go.